app/services/tock_importer.rb

```diff
@@ -6,6 +6,7 @@ def perform
       tock_project = ClientAccount.find_or_initialize_by(tock_id: project["id"])
       tock_project.name = project["name"]
       tock_project.billable = project["billable"]
+      tock_project.live = project["live"]
```

Comment:
more than this? https://github.com/18F/micropurchase/blob/cea84c1dad7f1c11006005bae860f22f7c740065/spec/services/tock_importer_spec.rb